src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    operate(callback, value) {
+      callback(value);
+
+      return calculator;
+    },
+    add(value) {
+      result += value;
+
+      return calculator;
+    },
+    reset() {
+      result = 0;
+
+      return calculator;
+    },
+    subtract(value) {
+      result -= value;
+
+      return calculator;
+    },
+    divide(value) {
+      if (value !== 0) {
+        result /= value;
```

Comment:
In the divide function, you should throw an error when 'value' is 0 to avoid division by zero.